index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="icon" sizes="192x192" href="{% static 'img/ci-favicon.png' %}" type="image/x-icon">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+        integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
+        crossorigin="anonymous">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <title>Hackathon</title>
+
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <h1 class= "text-center col-12 col-sm-6 col-md-12">Upcoming Hackathon</h1>
+            <div class="card col-6 col-sm-12 col-md-6" style="width: 18rem;">
+                <div class="card-body">
+
+                    <div class="collapse-content">
+
+                    <!-- Text -->
```

Comment:
Could you please remove this and all unnecessary comments below (e.g. `<!-- Text -->`, `<!-- Button -->`)?

Reply:
removed all comments and spaces.